src/app/[chainId]/[address]/sections/LibrariesSection.tsx

```diff
@@ -0,0 +1,202 @@
+import React from "react";
+import { TransformationValues } from "@/types/contract";
+import CopyToClipboard from "../../../../components/CopyToClipboard";
+import { IoCheckmarkCircle, IoCloseCircle, IoWarning } from "react-icons/io5";
+
+interface LibrariesSectionProps {
+  compilation: {
+    compilerSettings?: {
+      libraries?: Record<string, string>;
+    };
+  };
+  runtimeValues?: TransformationValues;
+  creationValues?: TransformationValues;
+  chainId: string;
+  verificationStatus: Record<string, boolean>;
+}
+
+function LibraryVerificationStatus({ address, isVerified }: { address: string; isVerified: boolean }) {
+  return (
+    <span className="ml-2">
+      {isVerified ? (
+        <IoCheckmarkCircle
+          className="text-green-600 text-xl"
+          data-tooltip-id="global-tooltip"
+          data-tooltip-content={`${address} is verified on Sourcify`}
+        />
+      ) : (
+        <IoCloseCircle
+          className="text-red-600 text-xl"
+          data-tooltip-id="global-tooltip"
+          data-tooltip-content={`${address} is not verified on Sourcify`}
+        />
+      )}
+    </span>
+  );
+}
+
+export default async function LibrariesSection({
+  compilation,
+  runtimeValues,
+  creationValues,
+  chainId,
+  verificationStatus,
+}: LibrariesSectionProps) {
+  const compilerLinkedLibraries = Object.entries(compilation?.compilerSettings?.libraries || {}).reduce(
+    (acc, [filePath, libraries]) => {
+      Object.entries(libraries as unknown as Record<string, string>).forEach(([libName, address]) => {
```

Comment:
is this `as unknown as Record<string, string>` necessary?

Reply:
Doesn't seem to be necessary